src/components/Body/index.tsx

```diff
@@ -18,6 +23,26 @@ const Body: React.FunctionComponent<Partial<IToastMessage>> = ({
       <div className="courier__title">{title}</div>
       <div className="courier__body">{body}</div>
     </ContentContainer>
+    {primaryAction || secondaryAction ? (
+      <ActionsContainer>
+        {primaryAction && (
+          <div
+            className="courier__primary-action"
+            onClick={primaryAction.onClick}
+          >
+            {primaryAction.text}
+          </div>
+        )}
+        {secondaryAction && (
+          <div
+            className="courier__secondary-action"
+            onClick={secondaryAction.onClick}
```

Comment:
these onclicks need to support href. 

imagine we get a message from the backend

```
{
   title: "title",
   body: "body",
   data: {
        triggeredBy: "riley",
        deliveredTrackingUrl: "blah",
        clickTrackingUrl: "test",
        clickAction: "https://www,courier.com",
        clickActionText: "Click Me",
        secondaryClickAction: "https://www.closeme.com",
        secondaryClickActionText: "Close Me",
   }
}

```

Reply:
```
{
   title: "title",
   body: "body",
   data: {
        triggeredBy: "riley",
        deliveredTrackingUrl: "blah",
        clickTrackingUrl: "test",
        clickAction:  "somehref" || {
             href: "somehref",
             text: "some text",
        },
        secondaryAction: "somehref" || {
             href: "somehref",
             text: "some text",
        },
   }
}
```

Reply:
Done :)